src/js/components/TaskListItemComponent.jsx

```diff
@@ -66,7 +66,7 @@ var TaskListItemComponent = React.createClass({
       const name = getPortDecoration(props.portDefinitions, 0, ports[0]);
       return (
         <a className="text-muted"
-            href={`${scheme}://${task.host}:${ports[0]}`}
```

Comment:
does it work if the task has no specific ip address?

Reply:
yes, ipAddresses will have the host's IP in that case.

Reply:
ok!